CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## DFX
 
+### chore: change the default Internet Computer gateway domain to `icp0.io`
+By default, DFX now uses the `icp0.io` domain to connect to Internet Computer as opposed to using `ic0.app`. 
+Canisters communicating with `ic0.app` will continue to function nominally.
+
```

Comment:
Since this updates the asset canister wasm, please add a dependencies section to the changelog, along with the new module hash (`shasum -a 256 src/distributed/assetstorage.wasm.gz`).

Example from below
```
## Dependencies

### Frontend canister
- Module hash: 98863747bb8b1366ae5e3c5721bfe08ce6b7480fe4c3864d4fec3d9827255480
- https://github.com/dfinity/sdk/pull/2982
```